### DISCUSSION TITLE
Add separate "Send date/time" field

### DISCUSSION THREAD

**Discussion by scrobbleme** [hizzle-co/noptin#765] [Ideas]
Hello,

Currently the post date is used for the sending the newsletter.

This is quite dangerous, as this is not obvious. People tend to just click "publish" and the newsletter is send.

Better would be a 

- required
- extra field

This way people have to knowingly select a future date.

**Comment by picocodes** [comment]
Hi @scrobbleme,

This is a good idea. We'll add a pre-publish check similar to the one shown when publishing a blog post. This way, writers can decide when to send the email immediately or specify a later date.

**Comment by picocodes** [comment]
Fixed
